doc/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W --keep-going
```

Comment:
This is useful for local builds (I actually use these options in my local copy), even if currently it does not work on readthedocs.

Reply:
Yeah, I realized that I wanted it for my local builds as well, so I am sneaking it in here :)